spec/requests/apartments_spec.rb

```diff
@@ -9,25 +9,279 @@
+                    user_id: user.id
+                }
+            }
+    
+            post '/apartments', params: apartment_params
+            apartment = JSON.parse(response.body)
+            expect(response).to have_http_status(422)
+            expect(apartment['bedrooms']).to include "can't be blank"
+        end
+
+        it "cannot create an apartment" do 
+            apartment_params = {
+                apartment: {
+                    street: "Auto Mall Parkway", 
+                    unit: "113",
+                    city: "D'Iberville",
+                    state: "MS",
+                    square_footage: 1200,
+                    price: "1300",
+                    bedrooms: 2,
+                    pets: "cats and dogs",
+                    image: "https://media.apts247.info/ce/ce9560e9239c4c3b8369663b0a699279/floorplans/1196_CoconutPalm_504586.jpg",
+                    user_id: user.id
+                }
+            }
+        
+            post '/apartments', params: apartment_params
+            apartment = JSON.parse(response.body)
+            expect(response).to have_http_status(422)
+            expect(apartment['bathrooms']).to include "can't be blank"
+        end
+
+        it "cannot create an apartment" do 
+            apartment_params = {
+                apartment: {
+                    street: "Auto Mall Parkway", 
+                    unit: "113",
+                    city: "D'Iberville",
+                    state: "MS",
+                    square_footage: 1200,
+                    price: "1300",
+                    bedrooms: 2,
+                    bathrooms: 2,
+                    image: "https://media.apts247.info/ce/ce9560e9239c4c3b8369663b0a699279/floorplans/1196_CoconutPalm_504586.jpg",
+                    user_id: user.id
+                }
+            }
+    
+            post '/apartments', params: apartment_params
+            apartment = JSON.parse(response.body)
+            expect(response).to have_http_status(422)
+            expect(apartment['pets']).to include "can't be blank"
+        end
+
+        it "cannot create an apartment" do 
+            apartment_params = {
+                apartment: {
+                    street: "Auto Mall Parkway", 
+                    unit: "113",
+                    city: "D'Iberville",
+                    state: "MS",
+                    square_footage: 1200,
+                    price: "1300",
+                    bedrooms: 2,
+                    bathrooms: 2,
+                    pets: "cats and dogs",
+                    user_id: user.id
+                }
+            }
+    
+            post '/apartments', params: apartment_params
+            apartment = JSON.parse(response.body)
+            expect(response).to have_http_status(422)
+            expect(apartment['image']).to include "can't be blank"
+        end
+    end
```

Comment:
You request specs for the 422 error is all named the same thing in the it block.  Make sure the name of each test is specific to the thing your are checking.